lib/data/seeds/pame_country_stats.csv

```diff
@@ -1,4 +1,4 @@
-ISO3,pame_pa_marine_area,pame_pa_land_area,pame_percentage_pa_marine_cover,pame_percentage_pa_land_cover
+"iso3","pame_pa_marine_area","pame_pa_land_area","pame_percentage_pa_marine_cover","pame_percentage_pa_land_cover"

```

Comment:
the `"` don't make much difference so you can probably leave them, but yeah definitely the ISO3 is an issue.
I would suggest running the import of the stats locally before deploying so you can see issues in advance

Reply:
Will update the release manual with that advice, thank you.